site/content/integrate/apps/quick-start-go/_index.md

```diff
@@ -66,17 +66,20 @@ Create a file called `manifest.json` containing:
 ```json
 {
 	"app_id": "hello-world",
+    "version":"v0.8.0",
```

Comment:
Strictly speaking semver forbids using a prefix like `v`. That is the convention that also Zendesk and ServiceNow follow.
```suggestion
    "version":"0.8.0",
```